frontend/src/components/LeaveReview/ReviewModal.tsx

```diff
@@ -149,7 +150,6 @@ const ReviewModal = ({ open, onClose, setOpen, landlordId, onSuccess, toastTime
   const onSubmit = async () => {
     try {
       setSending(true);
-      const user = await getUser(true);
       if (!user) {
```

Comment:
I don't think we need this null check in the onSubmit function anymore since you added it to openReviewModal function and it shows a sign-in error toast when the user is null

Reply:
Good point!